### PR TITLE
steps with bullets

### PR DESCRIPTION
**Pull request by PablitoGS**

### CODE REVIEW

File: styles/components/steps.scss [hunk 1]
```diff
@@ -47,4 +48,45 @@ Markup:
...
+      }
+
+      &.active::before {
+        background-color: $theme-color-primary;
+  		}
+  	}
+
+    span {
+      display: none;
+    }
```
Comment (beatrizip) [inline styles/components/steps.scss:74]: Why? Accesibility?

File: styles/generics/_docs-index.scss [hunk 2]
```diff
@@ -112,6 +112,8 @@ $sgi-toolbar:                '2.20';
 $sgi-timeline:               '2.21';
 $sgi-messaging:              '2.22';
 $sgi-video:                  '2.23';
+$sgi-steps:                  '2.24';
+$sgi-steps-bullet:           '2.24.1.1';
```
Comment (nguasch) [inline styles/generics/_docs-index.scss]: components should be in alphabetical order

**Review by nguasch — APPROVED** [review]
👍

File: styles/components/steps.scss (hunk 1 above)
Comment (PablitoGS) [reply styles/components/steps.scss:74]: Yes, that's it

**Review by jcger — APPROVED** [review]

**Pull request merged**